### ISSUE TITLE
Source code treats `assert` as a function, while it is a statement

### ISSUE THREAD

**Issue by Helveg** [LFPy/LFPy#246]
I often see `assert(condition)` used. This is a potential cause for bugs as this is evaluated as `assert bool((condition))` and might always be evaluated to `True` once someone tries to convert the statement to an `assert condtion, message` statement. `assert(condition, message)` will then actually be evaluated as a 2 element tuple `assert bool((condition, message))` which will always be `True` and the actual evaluation of `condition` will be entirely ignored, nullifying your asserts.

`assert` is a statement and should not be syntactically confused with a function. The parser ignores that you "forget" a space between `assert(condition)` .

All occurences of `assert(condition)` should be replaced with `assert condition`

**Comment by Helveg** [comment]
Follow up, whenever in the source code `assert condition, message` should've been used this construct is used instead:

```python
try:
 assert(condition)
except AssertionError:
  raise AssertionError(message)
```

Obviously `assert condition, message` is preferred.

**Comment by espenhgn** [comment]
Yes, LFPy should conform better to PEP-3109 (https://www.python.org/dev/peps/pep-3109/). Many other style fixes have already been incorporated in the LFPykit working branch origin@2.2.dev0 (PR #226) which will be merged quite soon, so please use that for base of comparison.

**Comment by espenhgn** [comment]
Fixed by merging #226
